resources/cloud-config/admiral.yaml

```diff
@@ -1,8 +1,8 @@
 #cloud-config
 
-coreos: 
+coreos:
   etcd2:
-    proxy: on
+    discovery: https://discovery.etcd.io/59b91eba6a0705e55ff470943ee5c14a
```

Comment:
We can't use the same for different instances, we need to create it dynamically.